src/scss/grommet-core/_objects.accordion-panel.scss

```diff
@@ -0,0 +1,18 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+.grommet .#{$grommet-namespace}accordion-panel {
+
+  > .#{$grommet-namespace}box--clickable {
+    user-select: none;
+    -webkit-user-select: none;
+  }
+
+  .#{$grommet-namespace}heading,
+  .#{$grommet-namespace}paragraph {
+    max-width: $size-medium;
```

Comment:
I would think the max-width would be controlled by the outer context. This would allow accordions to be used in a wider variety of situations.